gui/gui/src/TGSpeedo.cxx

```diff
@@ -480,7 +511,8 @@ void TGSpeedo::DrawNeedle()
    Translate(80.0, angle, &xpk0, &ypk0);
    Translate(67.0, angle, &xpk1, &ypk1);
 
-   // compute x/y position of the peak mark
+   fMeanVal = GetMean();
```

Comment:
if fBufferSize > 0
{
fMeanVal = ...
}

Reply:
right, I agree

Reply:
I think just calling `GetMean()` is fine, it returns `fMeanVal` itself if `fBufferSize == 0`